### DISCUSSION TITLE
Should vue2 + composition-api + pinia work on chrome in versions before v55?

### DISCUSSION THREAD

**Discussion by eeedvisss** [vuejs/pinia#1118] [Help and Questions]
Hi,  

First thank you for this amazing Store for Vue!

We just started to convert our vuex3 store to pinia and noticed that we get errors when trying to launch the app on chrome in version 54 and under(in versions chrome v55+ everything works as we would expect):

![image](https://user-images.githubusercontent.com/20832646/156782787-026800be-c625-4fc1-b490-c171b54d4af3.png)

async/await was introduced in chrome v55.

But why are @vue/devtools-api methods being called in production directly with code that is not compatible with older browsers?
Should pinia work on browsers earlier than chrome v55 or maybe just our pinia setup is wrong?

Our environment:
- Vue-CLI 4.5.15
- vue 2.6.14
- pinia 2.0.11
- @vue/composition-api 1.4.9

This can be reproduced with a clean vue install.

Reproduction repository with a clean vue 2 app(pinia is initialised in src/App.vue):
https://github.com/eeedvisss/pinia-test

Thanks!

**Comment by Rengge** [comment]
I had the same problem. Did you solve it?

**Comment by posva** [comment]
The devtools are indeed meant for recent browsers and should be tree shaken during the build process. Specifically during the minification.

My guess here is some outdated dependencies related to the build (e.g. vue-cli) that are not treeshaking. Updating them should remove this problem
